servicecontrol/logging.md

```diff
@@ -10,18 +10,71 @@ redirects:
 
 ### Logging
 
-Instances of the ServiceControl service write logging information and failed message imports to the file system.  
-Before v1.9 the logging level by default was set to `Info`, however this level can be quite verbose, so from v1.9 the default logging level is now `Warn`, this level is also configurable by adding the following to the configuration file:
+Instances of the ServiceControl service write logging information and failed message import stack traces to the file system.  
```

Comment:
@johnsimons  Isn't that what the `failedimports` folder under the logs directory is for?  It contains the stacktraces for failed imports which is a form of logging. I"ve always thought this was little weird that we don't capture this in the DB.  It's something that we could report on.